imageregistry/v1/types.go

```diff
@@ -195,6 +210,16 @@ type ImageRegistryConfigStorageS3 struct {
 	// Optional, defaults to false.
 	// +optional
 	VirtualHostedStyle bool `json:"virtualHostedStyle"`
+	// trustedCA is a reference to a config map containing a CA bundle. The
+	// image registry and its operator use certificates from this bundle
+	// together with certificates from the system trust bundle to verify S3
```

Comment:
why combined, why not replace?  In most other spots, specifying a CA bundle means, "use exactly this" and not specifying a CA bundle means "use the system trust".  I prefer that because it gives someone the ability to distrust the system certs for a connection.